test/e2e/controller/missing_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package e2e
+package controller
```

Comment:
Can we rename the file name to something else? missing_test.go sounds like it includes the tests that were missing from test coverage :D

Reply:
`crash_test.go` ? That better?

Tests where the controller crashes for whatever reason? That sound better?

Reply:
Thanks, much better!

Reply:
SGTM! Will make the change.